rails/ordinals/de-AT.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+{
+  :"de-AT" => {
+    :ordinal => proc { |number| "#{number}." }
```

Comment:
The hashes for german can be harmonized to use use the shorter syntax, like for the other locales. E.g.:

```suggestion
  "de-AT": {
    ordinal: => proc { |number| "#{number}." }
```